packages/autorest.typescript/test/rlcIntegration/generated/bodyComplexRest/src/bodyComplexRestClient.ts

```diff
@@ -7,9 +7,8 @@ import { BodyComplexRestClient } from "./clientDefinitions";
 export default function createClient(
   options: ClientOptions = {}
 ): BodyComplexRestClient {
-  const baseUrl = options.baseUrl ?? "http://localhost:3000";
```

Comment:
Do we have any test case for this in swagger repo?

Reply:
ut no, other integration test or smoke test yes.

Reply:
I didn't find any `createClient` signature changes for multiple parameter cases

Reply:
I thought you were asking whether we have test for single url parameter in case of any regression error. 
add the integration test for multiple url parameters here https://github.com/Azure/autorest.typescript/pull/1563/files#diff-87bf906cad32684f3972ea059bae2326eab6bc4eeb2a1de7aa149e5789450dea